sdk/src/test/java/com/karhoo/sdk/api/network/adapter/SealedCoroutineCallAdapterFactoryTest.kt

```diff
@@ -1,74 +1,78 @@
-package com.karhoo.sdk.api.network.adapter
```

Comment:
not a fan of commenting out tests after what happened with the payment related ones. could we fix them instead?

Reply:
Brought back the unit tests